industry_benchmarks/utils/fix_networks.py

```diff
@@ -666,6 +681,13 @@ def parse_args(arg_list = None):
         help="Results JSON file(s)",
         required=True,
     )
+    parser.add_argument(
+        "--allow-missing",
+        help="If we should ignore partially complete or missing edges and fix the network "
+             "rather than fail by default this is `False` and will raise an error.",
```

Comment:
```suggestion
             "rather than fail. By default this flag is set to `False`, meaning the script will raise an error if results are incomplete.",
```